### ISSUE TITLE
Questions about some detailed settings

### ISSUE THREAD

**Issue by bitpeach** [LouisBarranqueiro/hexo-theme-tranquilpeak#484]
### Preface
- Thank you for reading this issue. I feel sorry to disturb you for that I am a newbie firstly using both `Hexo` and `tranquilpeak`. Thank you for the design of `hexo-theme-tranquilpeak` to benefit us. :thumbsup: :raised_hands:
- I was confused that I always failed to implement page functions after I referred to some tutorials. Maybe I cannot follow your tutorial correctly. :sob:  :joy_cat:
- Could you establish a chat group such as wechat group or telegram group? Because the fans of `hexo-theme-tranquilpeak` will discuss each other easily in the chat group.

### Configuration

 - **Operating system with version** : Windows 10 x64
 - **Node version**: 8.9.3
 - **Hexo version**: 3.6.0
 - **Hexo-cli version**: 1.1.0
 - **Tranquilpeak version**: original
 - **Site configuration file** (Optional):
 - **Theme configuration file** (Optional):
 - **Hexo plugin** (Optional): 
 
### Expected behavior
- Favicon setting.
- Table of content.
- Sharing options.
- RSS questions.
- Demo page setting.
- Latex questions.

#### (1) Favicon
I have already made a favicon and put it into the `assets/image/` directory of theme documents. And I supplemented the ico file name into the `favicon: ` field of config file `_config.yml` in the theme document. However, the favicon was still not presented on the label bar of the web page.

#### (2) Table of content
Commonly `toc` is applicable to most of usage. However, because of my job, the title may contain the number and the using `toc` may cause this situation as the following.  In my titles, there already has the number and the toc brings the redundant list number.

> 1.~~1 Level One Title~~
>  &emsp;1.~~1.1 Level Two Title~~
>  &emsp;1.~~1.2 Level Two Title~~
> 2.~~2 Level One Title~~
>  &emsp;2.~~2.1 Level Two Title~~
>  &emsp;2.~~2.2 Level Two Title~~

I have tried to revise the `toc.js` and still failed. I have noticed that the`tranquilpeak\layout\_partial\post.ejs` has actually set the `list_number: false`. However, I still get the wrong toc with the list number. 

The similar issue has been discussed in the issue #21.

#### (3) Sharing options
When I want to use other social platforms, I am confused of how to determine the sharing URL links of different social platforms.

#### (4) RSS questions
I followed the tutorial setting RSS on the Internet. First, `hexo-generator-feed` has been installed. Second, I revise the config file in the blog document. Finally, I revise the config file in the theme document. However,I was failed and still got the error such as `This XML file does not appear to have any style information associated with it`.

#### (5) Demo page setting
I followed the tutorial and put the thumbnail image into the defined path. But I don't know how to revise the CSS or HTML files to make a page like your demo page, which several excerpts or thumbnail images are shown on the home page.

#### (6) Latex
On the Internet, there are lots of blogs helping to set the latex support of hexo that really confuses me.

- [ ] Someone uses the `Mathjax`.
- [ ] Someone uses the `hexo-math`.
- [ ] Someone replaces `hexo-renderer-marked` by `hexo-renderer-kramed`.

I already found that there was an issue #148 about supporting mathjax and using `hexo-math`. But there was no details about how to handle with this theme. According to the tutorial from the Internet, I installed `hexo-math` and wrote it into the `plugin` field. However, I tried those methods and still got failed. 

I think that I may be lack of some skills or knowledge and I am not good at revising the CSS and HTML files. And I was confused by the relationships among those methods. :joy:

If a chat group can be built, that is very excellent.

Sorry to disturb you.
Best regards.
Bitpeach

**Comment by LouisBarranqueiro** [comment]
👋 @bitpeach,  we have a chat! 😄  That's one of the first thing displayed in the README. Come here: https://gitter.im/LouisBarranqueiro/hexo-theme-tranquilpeak.

**Comment by bitpeach** [comment]
New messages.
The questions above have been solved partially.

#### (1) Favicon
In the local computer debugging, the icon file of the favicon is located in the `tranquilpeak\source\assets\images\` and you should set the fields in the `_config.yml` of theme documents. We cannot see the favicon on the webpage tag when you are in the local debugging. However, keeping this way, you can see the favicon after you deploy that on the github pages or servers.

Hence, in the local debugging, it is normal that you can see no icons. Without the change of local settings, you can see that the favicon appears after you deploy the hexo codes to Internet platforms.

#### (2) Demo page setting
By following the tutorial and putting the thumbnail image into the defined path, we may see no thumbnail images in the local computer. However, after you deploy those to servers, those thumbnail pictures may appear.

#### (3) Latex
After I have several experiments, I use the `hexo-renderer-kramed` and `Mathjax` for showing Latex, without `hexo-math`. This is successful for github pages.
- Uninstall `hexo-renderer-marked` and `hexo-math`, if you have those.
- Install `hexo-renderer-kramed`.
- Revise three files because of avoiding the conflict between `Mathjax` and `Markdown`.  I am not sure that which one will affect this conflict. So I revise all, i.e. `hexo-renderer-kramed\lib\renderer.js`, `node_modules/kramed/lib/rules/inline.js` and `tranquilpeak\node_modules\marked\lib\marked.js`. The revisions on three files are about two places including `escape` and `em`. Note that the targeted files need revision and the methods can be referred as [this Chinese blog](https://www.cnblogs.com/Ai-heng/p/7282110.html) or [this English blog](http://searene.me/2016/10/01/Let-hexo-support-mathjax/).
- If your `Mathjax` is loaded slowly, you can refer to [this](https://www.mathjax.org/cdn-shutting-down/) to change the CDN links.

Thanks.
We need more interactive communications.
Best regards.

**Comment by bitpeach** [comment]
Thank you. The questions above are almost solved. Please close this issue. 

Thank you for your help.
